src/main/java/com/webonise/rbs/controller/IndexController.java

```diff
@@ -0,0 +1,16 @@
+package com.webonise.rbs.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class IndexController {
+
+    @GetMapping(value = "/")
```

Comment:
no need to specify "/". If you specify then you will get data on "www.abc/" not "www.abc"